mkosi/__init__.py

```diff
@@ -1542,6 +1543,13 @@ def run_selinux_relabel(state: MkosiState) -> None:
         run_workspace_command(state.root, ["sh", "-c", cmd], env=state.environment)
 
 
+def run_zero_mtime(state: MkosiState) -> None:
+    for p in state.root.glob('**/*'):
```

Comment:
We should use `complete_step()` to print a message here that we're zeroing mtimes.